01.fizzbuzz/fizzbuzz.rb

```diff
@@ -0,0 +1,16 @@
+a=1
+
+20.times{
+
+if a%15 == 0 
```

Comment:
```suggestion
if a % 15 == 0 
```

演算子の前後にはスペースを入れましょう。

それからブロックの内部はインデントを入れてください。

```ruby
20.times do
  if a % 15 == 0 
```